scripts/obtain/dotnet-install.sh

```diff
@@ -648,6 +655,8 @@ do
             echo "  --no-path, -NoPath             Do not set PATH for the current process."
             echo "  --verbose,-Verbose             Display diagnostics information."
             echo "  --azure-feed,-AzureFeed        Azure feed location. Defaults to $azure_feed"
+            echo "  --linux-portable               Installs the Linux portable shared runtime instead of the shared runtime for OS."
+			echo "      -LinuxPortable"
```

Comment:
Which line? -LinuxPortable? I was just copying what was there for -SharedRuntime. I think it's supposed to show it can also be used as the command-line argument. But I'll fix it to appear on the same line like -AzureFeed.